src/NHSD.BuyingCatalogue.Ordering.Api/Models/CreateOrderItemBaseModel.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public abstract class CreateOrderItemBaseModel
+    {
+        [Required(ErrorMessage = "ServiceRecipientRequired")]
+        public ServiceRecipientModel ServiceRecipient { get; set; }
+
+        [Required(ErrorMessage = "ProvisioningTypeRequired")]
+        public string ProvisioningType { get; set; }
+
+        [Required(ErrorMessage = "TypeRequired")]
+        public string Type { get; set; }
+
+        [Required(ErrorMessage = "CurrencyCodeRequired")]
+        public string CurrencyCode { get; set; }
+
+        [Required(ErrorMessage = "ItemUnitRequired")]
+        public ItemUnitModel ItemUnit { get; set; }
+
+        [Required(ErrorMessage = "QuantityRequired")]
+        [Range(1, int.MaxValue, ErrorMessage = "QuantityGreaterThanZero")]
+        public int? Quantity { get; set; }
+
+        public string EstimationPeriod { get; set; }
+
+        [Required(ErrorMessage = "PriceRequired")]
+        [Range(0, double.MaxValue, ErrorMessage = "QuantityGreaterThanZero")]
+        public decimal? Price { get; set; }
```

Comment:
The error message is referring to Quantity. I think it should be 'PriceGreaterThanOrEqualToZero'.

Reply:
ah good call!

Reply:
These updated validation statuses are a right pain. They weren't covered in the JSON contract, and are going to make things a little awkward. I think I'll have to write another couple of custom validation attributes to cover it... groovy